src/http/api/note.ts

```diff
@@ -62,24 +60,16 @@ export async function handleCreateNote(
 
     const post = getValue(postResult);
 
-    let result: ActivityJsonLd | null = null;
+    const account = ctx.get('account');
 
-    try {
-        result = await publishNote(ctx, {
-            content: post.content ?? '',
-            author: {
-                handle: ACTOR_DEFAULT_HANDLE,
-            },
-            apId: post.apId,
-            imageUrl: post.imageUrl,
-        });
-    } catch (err) {
-        ctx.get('logger').error('Failed to publish note: {error}', {
-            error: err,
-        });
-    }
+    const postDTO = postToDTO(post, {
+        authoredByMe: post.author.id === account.id,
```

Comment:
This is always true, right?

Reply:
Correct! Fixed in https://github.com/TryGhost/ActivityPub/pull/551/commits/4a4a080f43440366183d9ed983a77132f8a21e9a